src/style.css

```diff
@@ -1,3 +1,83 @@
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  --text-color: #000;
+  --text-color-active: #00acdc;
+  --header-height: 60px;
```

Comment:
Great that you used variables 🔥 
Note that it's a good practice to use :root selector for them
![image](https://github.com/mate-academy/layout_moyo-header/assets/96409563/e2d6ce56-0f82-441e-99d8-70045e382be3)